static/templates/task/base.html

```diff
@@ -1,4 +1,4 @@
-<div class="_main-content" style="width: 100%">
```

Comment:
@mbernst the issue is something else in this case: when he merged develop2 into his branch he discarded the new changes from remote server and accepted only his local changes